packages/components/src/components/typography/Link/Link.tsx

```diff
@@ -22,17 +22,19 @@ type AllowedLinkTextProps = Pick<TextPropsCommon, (typeof allowedLinkTextProps)[
 
 type AProps = TransientProps<AllowedLinkTextProps> & {
     $variant?: 'default' | 'nostyle' | 'underline';
+    $gap?: keyof SpacingPx;
+    $color?: string;
 };
 
 const A = styled.a<AProps>`
     text-decoration: none;
     cursor: pointer;
-    color: ${({ theme }) => theme.textDefault};
+    color: ${({ $color, theme }) => $color || theme.textDefault};
     font-weight: 500;
     display: inline-flex;
     align-items: center;
 
-    gap: ${spacingsPx.xxs};
+    gap: ${({ $gap }) => ($gap ? spacingsPx[$gap] : spacingsPx.xxs)};
```

Comment:
I think default should be set on the gap prop, not in the css, and it should be typed to SpacingValues.

Reply:
Thanks, did this